src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,237 @@
+---
+title: CI/CD for API Mesh
+description: Learn how to set up continuous integration and continuous delivery/deployment for API Mesh .
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# CI/CD for API Mesh
+
+API Mesh for Adobe Developer App Builder now provides CI/CD through [Github actions](https://docs.github.com/en/actions). This allows developers to build, test, and deploy meshes faster and more reliably.
+
+## API Mesh authentication
+
+<InlineAlert variant="info" slots="text"/>
+
+[Create your mesh](./create-mesh.md) before beginning this process, the supplied GitHub action only handles mesh updates.
+
+To get an OAuth token to enable CI/CD:
+
+1. Navigate to the [Adobe Developer Console](https://developer.adobe.com/console) and select your desired workspace.
```

Comment:
I think we should should say select the workspace where your mesh is currently, to avoid confusion